frontend-web/webclient/app/Project/Subprojects.tsx

```diff
@@ -556,7 +556,10 @@ const SubprojectRow: React.FunctionComponent<{
             left={
                 isEditingName && allowManagement ? (
                     <>
-                        <Input ref={subprojectRenamingRef} fontSize="large" defaultValue={subproject.title} />
+                        <Button pr={8} pl={8} disabled={isEditingName}>
```

Comment:
Couldn't you just write `disabled` here? You have already tested the `isEditingName` boolean above.